jinahub/indexers/storage/PostgreSQLStorage/tests/test_postgres_dbms.py

```diff
@@ -184,6 +184,51 @@ def test_return_embeddings(docker_compose):
     assert query2[0].embedding is None
 
 
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_get_documents(docker_compose):
+    indexer = PostgreSQLStorage()
+
+    NR = 10
+    docs = DocumentArray(
+        list(
+            get_documents(
+                nr=NR,
+                chunks=0,
+                same_content=False,
+            )
+        )
+    )
+
+    indexer.add(docs)
+    assert len(list(indexer.get_document_iterator())) == NR
+
+    indexer.delete(docs)
+    assert len(list(indexer.get_document_iterator())) == 0
+    assert indexer.size == 0
+
+
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_clear(docker_compose):
+    indexer = PostgreSQLStorage()
+
+    NR = 10
+    docs = DocumentArray(
+        list(
+            get_documents(
+                nr=NR,
+                chunks=0,
+                same_content=False,
+            )
+        )
+    )
+
+    indexer.add(docs)
+    assert len(list(indexer.get_document_iterator())) == NR
+
+    indexer.clear()
```

Comment:
Seeing this test, I wonder if right name here should just be `delete_all`

Reply:
`/clear` should have the same meaning as in other indexers.